frontend/public/components/pdb/pdb-models.ts

```diff
@@ -0,0 +1,43 @@
+import { MatchLabels } from '@console/dynamic-plugin-sdk/src/api/common-types';
+import { PodDisruptionBudgetKind } from '../../module/k8s';
+import { PodDisruptionBudgetModel } from '../../models';
+
+export const convertToInt = (val) => {
```

Comment:
This function name is a little deceptive since it doesn't always convert. Maybe something like `intOrPercentage`. Add types for `value` and return types.